src/lib/containers/user_profile_links/UserProjects.tsx

```diff
@@ -72,7 +74,10 @@ export default function UserProjects({ userId }: UserProjectsProps) {
         </>
       )}
       {!isFetching && allRows.length == 0 && <div>Empty</div>}
-      {isFetching && <SkeletonTable numRows={5} numCols={1} />}
+      {isLoading && <SkeletonTable numRows={5} numCols={1} />}
+      {isFetching && !isLoading && <div className="placeholder">
+        <SynapseSpinner size={30} />
+      </div>}
```

Comment:
Just my opinion: this spinner will pop up each time it re-fetches, and with our current config, React Query is very eager to refetch. I think for now we should omit this entirely (or we can consider changing our React Query config).

If we _need_ to tell the user that data is re/fetching, then I think we would want a more subtle indicator. We would probably want to talk to design about that.

Reply:
Works for me.  Spinners have been removed.